webapp/crashstats/signature/tests/test_views.py

```diff
@@ -2,506 +2,524 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import datetime
 import json
-from urllib.parse import quote
 from unittest import mock
+from urllib.parse import quote
 
 import pyquery
 
 from django.urls import reverse
 from django.utils.encoding import smart_str
 
 from crashstats.crashstats import models
-from crashstats.crashstats.tests.test_views import BaseTestViews
-from crashstats.supersearch.models import SuperSearchUnredacted
+from socorro.lib.libdatetime import date_to_string, utc_now
+from socorro.lib.libooid import create_new_ooid, date_from_ooid
 
 
-DUMB_SIGNATURE = "hang | mozilla::wow::such_signature(smth*)"
+TEST_SIGNATURE = "shutdownhang | js::NewProxyObject"
 
 
-class TestViews(BaseTestViews):
-    def setUp(self):
-        super().setUp()
-        # Mock get_versions_for_product() so it doesn't hit supersearch breaking the
-        # supersearch mocking
-        self.mock_gvfp = mock.patch(
-            "crashstats.crashstats.utils.get_versions_for_product"
-        )
-        self.mock_gvfp.return_value = ["20.0", "19.1", "19.0", "18.0"]
-        self.mock_gvfp.start()
+def get_date_range(crash_id=None, date=None):
```

Comment:
The supersearch only returns items with a certain date range, so we need to make sure the requests made by the tests use a date range that works for the crash reports it indexed. This function makes that easier.